1.wave2_data/1.cellprofiler_ic_processing/README.md

```diff
@@ -0,0 +1,42 @@
+# Create initial LoadData csv to calculate illumination correction function
+
+In this module, we create a LoadData csv and CellProfiler pipeline for calculating an illumination correction function for each channel.
```

Comment:
Consider adding an abbreviation indication for illumination correction here so it may be appropriately used in abbreviated form later in the document.
```suggestion
In this module, we create a LoadData csv and CellProfiler pipeline for calculating an illumination correction (IC) function for each channel.
```

Reply:
Great idea!